hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -86,8 +87,9 @@
     versionProvider = HddsVersionProvider.class,
     mixinStandardHelpOptions = true,
     showDefaultValues = true)
+@MetaInfServices(FreonSubcommand.class)
 @SuppressWarnings("java:S2245") // no need for secure random
-public final class RandomKeyGenerator implements Callable<Void> {
+public final class RandomKeyGenerator implements Callable<Void>, FreonSubcommand {
```

Comment:
Looks like to me that RandomKeyGenerator should be rewritten by extending BaseFreonGenerator.
Of course that's a much larger rewrite that's out of the scope here.